corehq/apps/registration/tests/test_utils.py

```diff
@@ -0,0 +1,60 @@
+from django.contrib.auth.models import User
+from django.test import TestCase, RequestFactory
+
+from corehq.apps.accounting.models import Subscription
+from corehq.apps.domain.models import Domain
+from corehq.apps.registration.utils import request_new_domain
+from corehq.apps.users.models import WebUser
+
+
+class TestRequestNewDomain(TestCase):
```

Comment:
Awesome tests. It's unfortunate that we have to create things like `request` to be able to test this code, but I think the tests isolate that out well and are straight-to-the-point